lte/gateway/python/integ_tests/s1aptests/test_attach_detach_rar_activation_reject.py

```diff
@@ -0,0 +1,133 @@
+"""
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import unittest
+import s1ap_types
+import time
+
+from integ_tests.s1aptests import s1ap_wrapper
+from integ_tests.s1aptests.s1ap_utils import SessionManagerUtil
+
+
+class TestAttachDetachRarActivationReject(unittest.TestCase):
+    def setUp(self):
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+        self._sessionManager_util = SessionManagerUtil()
+
+    def tearDown(self):
+        self._s1ap_wrapper.cleanup()
+
+    def test_attach_detach_rar_activation_reject(self):
+        """ attach/detach + rar + dedicated bearer activation reject test
+            with a single UE """
+        num_ues = 1
+        detach_type = [
```

Comment:
As num_ues is 1, only UE_NORMAL_DETACH will be used in this test. We can remove the other option, unless we are planning to add more UEs in this test.

Reply:
I found it useful to parameterize num of UEs.